test/actions/katello/organization_test.rb

```diff
@@ -144,6 +144,7 @@ class ManifestRefreshTest < TestBase
       rhel7 = katello_repositories(:rhel_7_x86_64)
       acme_org.stubs(:owner_details).returns({})
       acme_org.products.stubs(:redhat).returns([rhel7.product])
+      acme_org.stubs(:simple_content_access?).returns(false)
```

Comment:
Since we're testing an action class I think it'd be beneficial to return `true` for your stub here, and actually execute your method call! (ditto for other actions)

Reply:
Changed to true on upload/refresh/delete. Tests pass and we are now testing my method :)